oak/server/storage/storage_node.cc

```diff
@@ -78,26 +78,32 @@ void StorageNode::Run(Handle request_handle) {
         // TODO: Handle errors.
       }
       StorageChannelReadResponse channel_read_response;
-      storage_processor_.Read(
-          channel_read_request.storage_name(), channel_read_request.datum_name(),
-          channel_read_request.transaction_id(), channel_read_response.mutable_datum_value());
-
+      std::string value;
+      storage_processor_.Read(channel_read_request.storage_name(),
+                              channel_read_request.datum().name(),
+                              channel_read_request.transaction_id(), &value);
+      channel_read_response.mutable_datum()->ParseFromString(value);
+      // TODO: Check security policy for datum.
```

Comment:
Could you create an issue for this and link it as `TODO(#xxx)` here?

Reply:
Done.